packages/eslint-plugin-lit-a11y/tests/lib/rules/interactive-supports-focus.js

```diff
@@ -0,0 +1,129 @@
+  { code: 'html`<a @click=${() => void 0} />`' },
+  { code: 'html`<a tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<a tabIndex=${dynamicTabIndex} @click=${() => void 0} />`' },
+  { code: 'html`<a tabIndex=${0} @click=${() => void 0} />`' },
+  { code: 'html`<a role="button" href="#" @click=${() => void 0} />`' },
+  { code: 'html`<a @click=${() => void 0} href="http://x.y.z" />`' },
+  { code: 'html`<a @click=${() => void 0} href="http://x.y.z" tabIndex="0" />`' },
+  { code: 'html`<a @click=${() => void 0} href="http://x.y.z" tabIndex=${0} />`' },
+  // TODO: { code: 'html`<a @click=${() => void 0} href="http://x.y.z" role="button" />`' },
+  { code: 'html`<foo-bar @click=${doFoo}><foo-bar/>`' },
+  { code: 'html`<input @click=${() => void 0} type="hidden" />`' },
+  { code: 'html`<span @click="${submitForm()}">Submit</span>`' },
+  { code: 'html`<span @click="${submitForm()}" tabIndex=${undefined}>Submit</span>`' },
+  { code: 'html`<span @click="${submitForm()}" tabIndex="bad">Submit</span>`' },
+  { code: 'html`<span @click="${doSomething()}" tabIndex="0">Click me!</span>`' },
+  { code: 'html`<span @click="${doSomething()}" tabIndex=${0}>Click me!</span>`' },
+  { code: 'html`<span @click="${doSomething()}" tabIndex="-1">Click me too!</span>`' },
+  {
+    code: 'html`<a href="javascript:void(0);" @click="${doSomething()}">Click ALL the things!</a>`',
+  },
+  { code: 'html`<section @click=${() => void 0} />`' },
+  { code: 'html`<main @click=${() => void 0} />`' },
+  { code: 'html`<article @click=${() => void 0} />`' },
+  { code: 'html`<header @click=${() => void 0} />`' },
+  { code: 'html`<footer @click=${() => void 0} />`' },
+  { code: 'html`<div role="button" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="checkbox" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="link" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="menuitem" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="menuitemcheckbox" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="menuitemradio" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="option" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="radio" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="spinbutton" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="switch" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="tablist" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="tab" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="textbox" tabIndex="0" @click=${() => void 0} />`' },
+  { code: 'html`<div role="textbox" aria-disabled="true" @click=${() => void 0} />`' },
+  { code: 'html`<foo-bar @click=${() => void 0} aria-hidden=${false}><foo-bar/>`' },
+  { code: 'html`<Input @click=${() => void 0} type="hidden" />`' },
```

Comment:
this `Input` looks like a react component 🙂

Reply:
🙈